File: Modul-2/Soal-3/PRAK203-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int a, b, i, j, x, y;
    float hasil;

    printf("Masukan Nilai a, b, i, j, x , y :");
    scanf(" %d %d %d %d %d %d", &a, &b, &i, &j, &x, &y);


    hasil = ((a-b)*(i*1.f/j)-(x+y));
    printf("Hasil dari %d dikurang %d dikali dengan %d dibagi %d dikurang dengan %d ditambah %d adalah %.3f",a,b,i,j,x,y,hasil);
}

File: Modul-2/Soal-3/PRAK203-2410817110016-MuhammadAzriRahman.py
total_input = []
print("Masukan Nilai a, b, i, j, x , y: ")
while len(total_input) < 6:
     nilai = input("")
     total_input.extend(nilai.split())

a, b, i, j, x , y = total_input
a, b, i, j, x, y = map(int, (a, b, i, j, x ,y))
hasil = (a-b)*(i/j)-(x+y)

print(f"Hasil dari {a} dikurang {b} dikali dengan {i} dibagi {j} dikurang dengan {x} ditambah {y} adalah {hasil:.3f}")